scipy/stats/morestats.py

```diff
@@ -1816,6 +1838,13 @@ def rootfunc(ab, xj, N):
         logsf = distributions.gumbel_r.logsf(w)
         sig = array([25, 10, 5, 2.5, 1])
         critical = around(_Avals_gumbel / (1.0 + 0.2/sqrt(N)), 3)
+    elif dist == 'weibull_min':
+        c, loc, scale = distributions.weibull_min.fit(y, loc=0.0)
```

Comment:
I propose to write `m, loc, ...`. otherwise it is very confusing since the paper uses m and c = 1/m, especially if we write above that "Keys are c*100"

Reply:
Done.